package.json

```diff
@@ -61,7 +61,9 @@
   },
   "peerDependencies": {
     "react": "^16.8 || ^17 || ^18",
-    "react-native": ">=0.71.6"
+    "react-native": ">=0.71.6",
+    "react-native-gesture-handler": "*",
+    "react-native-reanimated": "*"
```

Comment:
At first, react-native-gesture-handler is not necessary anymore, sorry.

I was just not confident to which extent this has meaning, because the success of the build depends on the versions of react-native-reanimated and other dependencies related to react-native.

But it is better to use "react-native-reanimated": ">=3.0.0" to show this library doesn't work 100% out of this range at least,So I will fix this.

Reply:
I adopted 3.2.0 with reference to [this](https://docs.swmansion.com/react-native-reanimated/docs/guides/compatibility/)